java/java.disco/src/org/netbeans/modules/java/disco/QuickPkgSelection.java

```diff
@@ -57,6 +59,9 @@ public QuickPkgSelection(QuickPanel.QuickSelection quick) {
                     switch (OS.getOperatingSystem()) {
                         case WINDOWS:
                             return ArchiveType.MSI == p.getArchiveType() || ArchiveType.EXE == p.getArchiveType();
+                        case LINUX:
```

Comment:
Finally getting a chance to look through these changes. This doesn't look right?  Should be returning things like DEB and RPM? However, how do we return the right one of those anyway?  I do wonder if "installers" should be left for the advanced panel and we remove that option entirely here?

Reply:
Well, I had started on sketching a UI rewrite (which I'll now shelve most of) but I had planned to remove this option from the quick panel, and still think we should do so. Considering the point of this wizard is to register a platform, the download installer option doesn't actually allow for that. I think we should keep the quick panel to download and register only.

Reply:
this would simplify things for sure. looking through the classes I noticed there were attempts to use the shell and an external process to extract the files. I thought maybe its ultimately planned to install things the same way.
But i am a linux user - and never install JDKs (or NetBeans) personally so i don't really care about this feature ;)

Reply:
Back looking at this.  Yes, the shell extraction appears mainly to be about execution permissions, judging from [this comment](https://github.com/apache/netbeans/blob/master/java/java.disco/src/org/netbeans/modules/java/disco/archive/JDKCommonsUnzip.java#L32). That can't work as it misses out `jspawnhelper` at least. I'm working on porting the extraction and detection solution from NBPackage.

Also a Linux user, and don't see the utility for installing the JDK here.  OTOH, I do like my NetBeans as a DEB! :wink:

Reply:
@neilcsmith-net feel free to do a second pass to correct things or write me a mail to me with a list you want changed. This PR started out as me wanting to simply add another column to a table - but ended up changing a lot of other things and re-doing most of the layouts in the form designer.

I am all for removing code btw, if the shell extraction code isn't needed anymore - lets remove it (I thought there were more plans for it).